src/ranged.cpp

```diff
@@ -32,6 +32,7 @@ const skill_id skill_throw( "throw" );
 const skill_id skill_gun( "gun" );
 const skill_id skill_melee( "melee" );
 
+static inline projectile make_gun_projectile( const item &gun );
```

Comment:
Yes, I suppose any recent compiler has rather reduced it to a statement of intent rather than fact.